antarest/launcher/service.py

```diff
@@ -238,13 +263,24 @@ def run_study(
         study_uuid: str,
         launcher: str,
         launcher_parameters: LauncherParametersDTO,
+        launcher_configuration_id: Optional[str] = None,
         study_version: Optional[str] = None,
     ) -> str:
         job_uuid = self._generate_new_id()
         logger.info(f"New study launch (study={study_uuid}, job_id={job_uuid})")
         study_info = self.study_service.get_study_information(uuid=study_uuid)
         solver_version = SolverVersion.parse(study_version or study_info.version)
 
+        if launcher_configuration_id is not None:
+            launcher_config = self.get_launcher_config(launcher_configuration_id)
+            if not is_launcher_config_compatible(
+                launcher_config, StudyVersion.parse(study_version or study_info.version)
+            ):
+                raise IncompatibleLauncherConfig("Launcher configuration is not compatible with study version")
+            if launcher_parameters.other_options:
```

Comment:
I think that in that case, what the user would want is to concatenate with those "other_options". However, It's actually not that easy to implement, I think, because we should do some merging of options, etc ...

I think in a first version we should keep it this way.